experiments/standalone/Bucket/global_bucket_staticmap.jl

```diff
@@ -1,7 +1,7 @@
 # # Global bucket run using spatial map albedo
```

Comment:
Should we make a different file for a regional experiment? 
Since this file is called "global_ ..."

(or rename this file)

Reply:
I renamed this to `bucket_era5` because `staticmap` was misleading